File: Exercícios02/ex08.c
#include <stdio.h>

int main(void){
    int N1, N2;

    printf("Digite os valores de N1 e N2: ");
    scanf("%d %d", &N1, &N2);

    if (N1 < N2){
        printf("Todos os números entre %d e %d:\n", N1, N2);
        for(int i = N1 + 1; i < N2; i++){
            printf("%d\n",i);
        }
    } else {
        printf("Todos os números entre %d e %d:\n", N2, N1);
        for(int i = N2 + 1; i < N1; i++){
            printf("%d\n",i);
        }
    }
    return 0;
}

File: Exercícios02/ex08.py
N1 = int(input("Digite o valor de N1: "))
N2 = int(input("Digite o valor de N2: "))

if N1 < N2:
    print(f"Todos os números entre {N1} e {N2}:")
    for i in range(N1+1,N2):
        print(f"{i}")
else:
    print(f"Todos os números entre {N2} e {N1}:")
    for i in range(N2+1,N1):
        print(f"{i}")